app/models/miq_server.rb

```diff
@@ -122,6 +122,10 @@ def validate_is_deleteable
     end
   end
 
+  def server_monitor
+    @server_monitor ||= ServerMonitor.new(self)
```

Comment:
NOTE I started following the pattern of `WorkerManagement` but the existing code did a `reload` every time it was called, not sure keeping an instance around buys us much

Reply:
I think it's good - it sets up the pattern that there should be a singleton instance of it, regardless of what it does inside.